hubploy/auth.py

```diff
@@ -72,28 +82,33 @@ def registry_auth_aws(deployment, project, zone, service_key):
         raise FileNotFoundError(
             f'The service_key file {service_key_path} does not exist')
 
-    # move credentials to standard location
-    cred_dir = os.path.expanduser('~/.aws')
-    if not os.path.isdir(cred_dir):
-        os.mkdir(cred_dir)
-    shutil.copyfile(service_key_path, os.path.join(cred_dir, 'credentials'))
-
-    registry = f'{project}.dkr.ecr.{zone}.amazonaws.com'
-    # amazon-ecr-credential-helper installed in .circleci/config.yaml
-    # this adds necessary line to authenticate docker with ecr
-    docker_config_dir = os.path.expanduser('~/.docker')
-    os.makedirs(docker_config_dir, exist_ok=True)
-    docker_config = os.path.join(docker_config_dir, 'config.json')
-    if os.path.exists(docker_config):
-        with open(docker_config, 'r') as f:
-            config = json.load(f)
-    else:
-        config = {}
+    original_credential_file_loc = os.environ.get("AWS_SHARED_CREDENTIALS_FILE", "")
```

Comment:
Let's use `None` as the default, not ""? IMO that's good practice for fiddling with env vars temporarily - we *do* want to differentiate between empty value "" and a marker that it did not exist

Reply:
Okay. I can change that and the unsetting logic so that it looks for `None`.